node_ops/task_manager_module.cc

```diff
@@ -91,7 +91,7 @@ tasks::task_manager::task_group node_ops_virtual_task::get_group() const noexcep
 future<std::set<tasks::task_id>> node_ops_virtual_task::get_ids() const {
     db::system_keyspace& sys_ks = _ss._sys_ks.local();
     service::topology& topology = _ss._topology_state_machine._topology;
-    co_return boost::copy_range<std::set<tasks::task_id>>(co_await get_entries(sys_ks, topology) | boost::adaptors::map_keys);
+    co_return boost::copy_range<std::set<tasks::task_id>>(co_await get_entries(sys_ks, topology, get_task_manager().get_task_ttl()) | boost::adaptors::map_keys);
```

Comment:
Is this filtering there to simulate tasks being removed after task TTL, similar to regular tasks?

Reply:
Yes, I'm not sure if this is useful though